miranda/io/utils.py

```diff
@@ -90,9 +95,14 @@ def name_output_file(
     if missing:
         raise ValueError(f"The following facets were not found: {' ,'.join(missing)}.")
 
-    return "{variable}_{frequency}_{institution}_{project}_{time_start}-{time_end}.{suffix}".format(
-        **facets
-    )
+    if project == "NEX-GDDP-CMIP6":
+        return "{bias_adjust_project}_{variable}_{frequency}_{institution}_{source}_{member}_{experiment}_{domain}_{time_start}-{time_end}.{suffix}".format(
+            **facets
+        )
+    else:
+        return "{variable}_{frequency}_{institution}_{project}_{time_start}-{time_end}.{suffix}".format(
+            **facets
+        )
```

Comment:
It would be good to have string definitions somewhere, maybe in a YAML/JSON file for each project? This would require some changes, but would open us up to more options/portability.

Reply:
Addressed in https://github.com/Ouranosinc/miranda/pull/120/commits/3a37f5ea8c6428a7cf2fe1b241a858a142fa67fd